circle.yml

```diff
@@ -1,9 +1,14 @@
 machine:
   python:
     version: 3.5.1
+  services:
+    - redis
 dependencies:
   pre:
     - pip install --upgrade pip
+    - pip install --upgrade virtualenv
+    - sudo add-apt-repository ppa:fkrull/deadsnakes-python2.7 -y
```

Comment:
Is this still needed since we want to use Xenial as a base box in CircleCI (I might be lacking context, so this is a genuine question).